src/components/views/admin/BuilderView.tsx

```diff
@@ -1,156 +1,243 @@
-                  top: '250px',
-                  left: '150px',
-                  borderRadius: '4px',
-                  backgroundColor: '#fff',
-                  boxShadow:
-                    '0px 1px 1px rgba(0, 0, 0, 0.14), 0px 2px 1px rgba(0, 0, 0, 0.12), 0px 1px 3px rgba(0, 0, 0, 0.2)',
-                  width: '800px',
-                  height: '650px',
-                  display: 'flex',
-                  flexDirection: 'column',
-                  alignItems: 'flex-start',
-                  justifyContent: 'flex-start',
-                  textAlign: 'center',
-                  fontSize: '20px',
-                }}
-              >
-                <p>CV HERE</p>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-      <div
-        style={{
-          position: 'absolute',
-          top: 'calc(50% - 147px)',
-          left: 'calc(50% - 166px)',
-          width: '614px',
-          height: '538px',
-        }}
-      >
-        <div
-          style={{
-            position: 'absolute',
-            top: '-150px',
-            left: '-450px',
-            letterSpacing: '0.1px',
-            color: '#6200ee',
-            display: 'flex',
-            alignItems: 'center',
-            width: '251px',
-            height: '52px',
-          }}
+    <Box>
+      <Box m={2} p={1} border="1px solid gray" borderRadius={2}>
+        <Box m={2} p={1} border="1px solid gray" borderRadius={2}>
+          <Box display="flex" alignItems="center">
+            <Typography>Download All Candidates CVs</Typography>
+            <IconButton onClick={() => handleDownloadAll()}>
+              <DownloadIcon />
+            </IconButton>
+          </Box>
+          {Object.keys(candidatesCv).map((memberId) => (
+            <Box key={memberId} display="flex" alignItems="center">
+              <Typography flex={1}>{memberId}</Typography>
+              <IconButton onClick={() => handleDownload(memberId)}>
+                <DownloadIcon />
+              </IconButton>
+              <IconButton onClick={() => handleView(memberId)}>
+                <VisibilityIcon />
+              </IconButton>
+              <Rating
+                name={`rating-${memberId}`}
+                value={ratings[memberId] || 0}
+                onChange={(event, newValue) =>
+                  handleRatingChange(memberId, newValue)
+                }
+                size="medium"
+              />
+            </Box>
+          ))}
+        </Box>
+        <Select
+          value={selectedType}
+          onChange={(e) => handleTypeChange(e.target.value)}
+          displayEmpty
+          sx={{ mr: 2 }}
         >
-          My Items/ CV App - Reviewer Interface
-        </div>
-      </div>
-    </div>
+          <MenuItem value="" disabled>
+            Select Type
+          </MenuItem>
+          {dimensions.map((option) =>
+            option.value === 'cvStatusInfo' ? null : (
```

Comment:
If you want to exclude something from your drop-down I think it would be better to simply not include it in the `dimensions`. What will you do when you have 2 keys you want to exclude ?

Reply:
Yh you're right, but if I wanted to structure the of the CV object for a candidate where is cvStatusInfo object is important to determine if the candidate have uploaded their own or used a template. And that's what made me do it in this way.